networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,16 @@ The Network Observability Operator is now stable and the release channel is upgr
 === Bug fix
 
 * Previously, unless the Loki `authToken` configuration was set to `FORWARD` mode, authentication was no longer enforced, allowing any user who could connect to the {product-title} console in an {product-title} cluster to retrieve flows without authentication.
-Now, regardless of the Loki `authToken` mode, only cluster administrators can retrieve flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
\ No newline at end of file
+Now, regardless of the Loki `authToken` mode, only cluster administrators can retrieve flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
+
+[id="network-observability-operator-release-notes-1-2"]
+== Network Observability Operator 1.2.0
+
+The following advisory is available for the Network Observability Operator 1.2.0:
+
+* <errata link here>
```

Comment:
Is the link missing?

Reply:
@lahinson The link is not available yet. I wanted to get this peer reviewed then all I have to do is add the link and merge on the day of GA. I probably should've noted that before review.